docs/topics/serialization.txt

```diff
@@ -268,6 +268,15 @@ work::
                 return force_text(obj)
             return super(LazyEncoder, self).default(obj)
 
+By default, :class:`~django.core.serializers.json.DjangoJSONEncoder` is used
```

Comment:
Using ``.. class:: django.core.serializers.json.DjangoJSONEncoder`` caused a line saying ``*class* django.core.serializers.json.DjangoJSONEncoder`` to just hang out in the middle of nowhere it seemed.  I couldn't find a way to define it as a :class: property without sphinx causing a line to be output there.  So I defined it as a regular :ref:, which seems hacky, particularly if it were to be linked to from anywhere outside of this little block of text.  Any suggestions are welcome.